<launch>
    
  <include file="$(find autorally_core)/launch/hardware.machine" />

    <node pkg="imu_3dm_gx4" name="imu" type="imu_3dm_gx4" output="screen" machine="autorally-master">
        <param name="device" type="string" value="/dev/arLordIMU" />
        <param name="baudrate" type="int" value="921600" />
        <param name="frame_id" type="string" value="imu"/>
        <param name="imu_rate" type="int" value="200" />
        <param name="filter_rate" type="int" value="200"/>
        <param name="enable_filter" type="bool" value="true"/>
        <param name="enable_accel_update" type="bool" value="true"/>
        <param name="enable_mag_update" type="bool" value="false"/>
        <param name="enable_gps_time_sync" type="bool" value="true"/>
        <param name="gps_time_fudge_factor" type="int" value="0"/>
    </node>
</launch>
